test/boost/sstable_compaction_test.cc

```diff
@@ -3851,7 +3852,7 @@ SEASTAR_TEST_CASE(test_offstrategy_sstable_compaction) {
             auto mut = mutation(s, pk);
             ss.add_row(mut, ss.make_ckey(0), "val");
 
-            table_for_tests cf(env.manager(), s, tmp.path().string());
```

Comment:
It looks like tmp is the back() element of a vector which doesn't have to be a vector -- test just creates new tempdir on next loop iteration and doesn't mess with previous one.